src/app/my-appliances/components/deployments/deployment/deployment.component.ts

```diff
@@ -16,7 +16,7 @@ import { DeploymentService } from '../../../../shared/services/deployment.servic
 import { MatDialog } from '@angular/material';
 import { ArchiveDeleteConfirmDlgComponent } from '../dialogs/archive-delete-confirm.component';
 
-const AUTOMATIC_HEALTH_CHECK_MINUTES = 7;
+const AUTOMATIC_HEALTH_CHECK_MINUTES = 0.5;
```

Comment:
This is probably too frequent. The health check results in a celery task, so this can take a bit of time. The original 7 minute interval between checks is probably better.